src/logic/operate.js

```diff
@@ -0,0 +1,36 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result = Big(0);
+  const a = Big(numberOne);
+  const b = Big(numberTwo);
+
+  switch (operation) {
+    case '+':
+      result = a.plus(b);
+      break;
+
+    case '-':
+      result = a.minus(b);
+      break;
+
+    case 'X':
+      result = a.times(b);
+      break;
+
+    case '/':
+      result = a.div(b);
+      break;
+
+    case '%':
+      result = a.div(100);
+      break;
```

Comment:
It will be better to implement the logic of this case using the `mod` function as provided in the `big api`.  For example: 
`one.mod(two).toString()`. To learn more about this kindly follow this [link](https://mikemcl.github.io/big.js/#mod)